src/runner/Run_semgrep.ml

```diff
@@ -559,7 +559,7 @@ let targets_of_config (config : Runner_config.t)
  * the original rules passed via -rules, without the extract-mode rules).
  *)
 let extracted_targets_of_config (config : Runner_config.t)
-    (rule_ids : Rule.rule_id list) (extractors : Rule.extract_rule list) :
+    (extractors : Rule.extract_rule list) (all_rules : Rule.t list) :
```

Comment:
I haven't looked much at the code but I don't understand why to pass the `extractors` here when it's a subset of `all_rules` and it's easy to compute right where you need it.

Reply:
oops great point